test/unit/Heatmap/GMUHeatmapTileLayerTest.swift

```diff
@@ -0,0 +1,102 @@
+/* Copyright (c) 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+@testable import GoogleMapsUtils
+
+class GMUHeatmapTileLayerTest: XCTestCase {
+  
+  private var startPoints: [NSNumber]!
+  private var colorMapSize: UInt!
+  private var gradientColor: [UIColor]!
+  private var firstTestCoordinate: CLLocationCoordinate2D!
+  private var secondTestCoordinate: CLLocationCoordinate2D!
+  private var intensity: Float!
+  private var cameraLatitude: Double!
+  private var cameraLongitude: Double!
+  private var mapsAPIKey: String!
+  private var radius: UInt!
+  private var minimumZoomIntensity: UInt!
+  private var maximumZoomIntensity: UInt!
```

Comment:
All these properties seem to only be needed in the `testHeatMapTileLayerDataPoints` test method. I recommend moving these as local variables to that method.